runtime/zeitgeist/src/parameters.rs

```diff
@@ -288,6 +288,35 @@ parameter_types! {
     /// Pallet identifier, mainly used for named balance reserves. DO NOT CHANGE.
     pub const TreasuryPalletId: PalletId = PalletId(*b"zge/tsry");
 
+    // Bounties
+    /// The amount held on deposit for placing a bounty proposal.
+    pub const BountyDepositBase : Balance = 10 * BASE;
+    /// The delay period for which a bounty beneficiary need to wait before claim the payout.
+    pub const BountyDepositPayoutDelay : BlockNumber = 3 * BLOCKS_PER_DAY;
+
+    /// Bounty duration in blocks.
+    pub const BountyUpdatePeriod : BlockNumber = 14 * BLOCKS_PER_DAY;
+
+    /// The curator deposit is calculated as a percentage of the curator fee.
+    ///
+    /// This deposit has optional upper and lower bounds with `CuratorDepositMax` and
+    /// `CuratorDepositMin`.
+    pub const CuratorDepositMultiplier : Permill = Permill::from_percent(50);
+
+    /// Maximum amount of funds that should be placed in a deposit for making a proposal.
+    pub const CuratorDepositMax : Balance = 200 * BASE;
```

Comment:
Based on the example of Kusama, they us a relation of 5:1 between `CuratorDepositMax` and `BountyDepositBase`.

```suggestion
    pub const CuratorDepositMax : Balance = 500 * BASE;
```